### ISSUE TITLE
Initial release

### ISSUE THREAD

**Issue by jaysoo** [nrwl/nx-react-native#1]
The initial release will comprised of integration with core React Native features.

- [x] `nx bundle [app]` (offline bundle)
- [x]  `nx start [app]`  (start server)
- [x] `nx run-ios [app]` 
- [x] `nx run-android [app]` 
- [x] support for third-party native packages
- [x] support for workspace libs (JS/TS only)
- [x] jest support
- [ ] ~e2e test using detox~

**Comment by 0b3r** [comment]
Looking forward to this release! Going to use it the second it's ready! Thanks to all of you for doing this awesome stuff.

**Comment by trumbitta** [comment]
I'm so ready to use this :D

**Comment by nclarx** [comment]
Agree, this will be great!!

**Comment by moQuez** [comment]
🚀

**Comment by itsezc** [comment]
Looking forward to this!

**Comment by tienne** [comment]
:1st_place_medal:

**Comment by jaysoo** [comment]
We're getting close. A fix for the jest setup is coming today https://github.com/nrwl/nx-react-native/pull/6.

And we've verified workflows for ios and android (starting project, dev build, prod build). ETA is next week -- week of November 9th.

**Comment by wjaszczuk** [comment]
@jaysoo yay, I'm looking forward to seeing it in action!

**Comment by bostondevin** [comment]
Yaaaaaaaayyyyyy!!!!!!

**Comment by jaysoo** [comment]
I've just published a beta of this plugin. It should be compatible with any version of Nx 10.

```
# Add plugin
yarn add @nrwl/react-native@10.4.0-beta.2

# Generate app
nx g @nrwl/react-native:app my-app

# Start JS bundler
nx start my-app

# Run in ios/android
nx run-ios my-app
nx run-android my-app

# Build production bundle (android)
nx build-android my-app

# Build production bundle (ios)
# Not supported through CLI, but open xcode project and build in there.
```

**Comment by jaysoo** [comment]
I published `10.4.0` so please use that version. Please open an issue for any feedback or bugs.
